File: app/src/main/java/com/codepath/bestsellerlistapp/models/BestSellerBook.java
package com.codepath.bestsellerlistapp.models;

import com.google.gson.annotations.SerializedName;

public class BestSellerBook {

   @SerializedName("rank")
   public int rank;

   @SerializedName("title")
   public String title;

   @SerializedName("author")
   public String author;

   @SerializedName("book_image")
   public String bookImageUrl;

   @SerializedName("publisher")
   public String publisher;

   @SerializedName("description")
   public String description;

   @SerializedName("amazon_product_url")
   public String amazonUrl;
}

File: app/src/main/java/com/codepath/bestsellerlistapp/models/BestSellerBook.kt
package com.codepath.bestsellerlistapp.models

import com.google.gson.annotations.SerializedName

class BestSellerBook {
    @SerializedName("rank")
    var rank = 0

    @JvmField
    @SerializedName("title")
    var title: String? = null

    @JvmField
    @SerializedName("author")
    var author: String? = null

    @SerializedName("book_image")
    var bookImageUrl: String? = null

    @SerializedName("publisher")
    var publisher: String? = null

    @SerializedName("description")
    var description: String? = null

    @SerializedName("amazon_product_url")
    var amazonUrl: String? = null
}